judges/quality-of-service/quality-of-service.rb

```diff
@@ -155,8 +155,8 @@
       "repo:#{repo} type:pr is:merged closed:>#{f.since.utc.iso8601[0..9]}"
     )[:items].each do |pr|
       pr_reviews = Fbe.octo.pull_request_reviews(repo, pr[:number])
-      pr_review = pr_reviews.min_by { |r| r[:submitted_at] }
-      review_times << (pr[:merged_at] - pr_review[:submitted_at]).to_i if pr_review
+      pr_review = pr_reviews.reject { |r| r[:submitted_at].nil? }.min_by { |r| r[:submitted_at] }
```

Comment:
@Yegorov what do you think about the using `filter_map`? 
```rb
pr_reviews.filter_map { |r| r if r[:submitted_at] }
```

Reply:
@Suban05 if use `filter_map` i get Rubocop warning
```
judges/quality-of-service/quality-of-service.rb:158:30: C: [Correctable] Style/MapCompactWithConditionalBlock: Replace filter_map { ... } with select.
      pr_review = pr_reviews.filter_map { |r| r if r[:submitted_at] }.min_by { |r| r[:submitted_at] }
```
As an option, may be use `select`? What do you think?
```ruby
pr_review = pr_reviews.select { |r| r[:submitted_at] }.min_by { |r| r[:submitted_at] }
```

Reply:
@Yegorov good idea, let's use `select`

Reply:
@Suban05 thanks, fixed